competition_inputs/BottleJSON.json

```diff
@@ -0,0 +1,32 @@
+{
+    "Bottle1": [
+	{"Letter": "M",
+	"Letter_Color": "Green",
+	"Shape": "Square",
+	"Shape_Color": "White" }
+    ],
+    "Bottle2": [
+	{"Letter": "E",
+	"Letter_Color": "Blue",
+	"Shape": "Square",
+	"Shape_Color": "White" }
+    ],
+    "Bottle3": [
+	{"Letter": "N",
+	"Letter_Color": "Red",
+	"Shape": "Square",
+	"Shape_Color": "White" }
+    ],
+    "Bottle4": [
+	{"Letter": "O",
+	"Letter_Color": "Yellow",
+	"Shape": "Square",
+	"Shape_Color": "White" }
+    ],
+    "Bottle5": [
+	{"Letter": "1",
+	"Letter_Color": "Blue",
+	"Shape": "Square",
+	"Shape_Color": "White" }
```

Comment:
At the competition, the ground station operator will have to load up this file with the actual values that the bottles will have. Since these are just example values, they can all be removed. For example, `"Letter": "O"` should become `"Letter": ""`